GitCommands/Git/GitModule.cs

```diff
@@ -595,6 +596,30 @@ public void SaveBlobAs(string saveAs, string blob)
             stream.Write(blobData, 0, blobData.Length);
         }
 
+        public void SaveTreeAs(string saveAs, string hash)
```

Comment:
The whole operation must be run via the `FormProcess`

Reply:
Thanks for the comments, I'm struggling a bit with this one. Running via FormProcess results in the dialog stealing focus, which interrupts the drag opperation, I'm not sure how to get around that?

Reply:
🤦‍♂️ of course! Silly me...